src/field/captcha/recaptcha.jsx

```diff
@@ -69,7 +96,8 @@ export class ReCAPTCHA extends React.Component {
   }
 
   reset() {
-    window.grecaptcha.reset(this.widgetId);
+    const global = globalForProvider(this.props.provider);
+    global.reset(this.widgetId);
```

Comment:
I would rename this in alignment with whatever you decide given my comment above.